splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/util/Validator.js

```diff
@@ -79,7 +79,7 @@ class Validator {
     }
 
     // Validate the custom component
-    CustomValidator(validatorFunc, field, data) {
+    static CustomValidator(validatorFunc, field, data) {
```

Comment:
When this method is called, you will have to use `className.CustomValidator` instead of `this` or initialized instance.

Reply:
That was eslint suggestions, as that method does not use **this** keyword, then that method should be static.
@dkhatri-crest  can you please confirm?

Reply:
@harshpatel-crest 
Yes, while calling we use classname.